woodstock/voting/models.py

```diff
@@ -28,6 +71,11 @@ class MozillianProfile(models.Model):
     avatar_url = models.URLField(max_length=400, default='')
     bio = models.TextField(blank=True, default='')
     username = models.CharField(max_length=100, default='')
+    application = models.ForeignKey(Application,
+                                    blank=True,
```

Comment:
Move ``blank``, ``null`` to the previous line. It looks like there is some space.